docs/source/cpp_api/neighbors_ivf_pq.rst

```diff
@@ -66,3 +66,11 @@ Index serialize
     :project: cuvs
     :members:
     :content-only:
+
+Helper Methods
```

Comment:
Can you add a small description for this section? For example, so that a user knows when they might want to use these functions.